runtime/onert/core/src/backend/basic/StaticTensorManager.cc

```diff
@@ -92,8 +95,17 @@ void StaticTensorManager::claimPlan(const ir::OperandIndex &ind, uint32_t size)
   // This method is called only when a tensor has proper shape
   assert(!_tensors->getNativeTensor(ind)->is_dynamic());
 
-  if (!_as_constants[ind])
-    _nonconst_mgr->claimPlan(ind, size);
+  const auto claim_ind = adjust_with_memory_source_operand(ind);
+  if (_as_constants[claim_ind])
+  {
+    return;
+  }
+  ++_source_operand_inds_ref_counter[claim_ind];
+  // notify only first usage
+  if (1 == _source_operand_inds_ref_counter[claim_ind])
+  {
+    _nonconst_mgr->claimPlan(claim_ind, size);
+  }
```

Comment:
I prefer that other tensors are planned without being affected by ref counter(`_source_operand_inds_ref_counter`). If you would like to use the ref counter, could you apply it only for memory sharing tensors?

Reply:
Good point. Now if we have [reassign_indexes_to_single_sources](https://github.com/Samsung/ONE/blob/master/runtime/onert/backend/cpu/SharedMemoryOperands.cc#L30) it seems to be safe